channels/email/advanced/fetchmail.rst

```diff
@@ -5,6 +5,10 @@ Maybe you want to add emails via Fetchmail or Procmail to Zammad.
 
 To get this to work you need to pipe your emails to rails.
 
+.. note:: Below commands might differ from system to system. If you want to learn on how 
+  to call Zammad commands on your system, please have a look on `our console introduction 
+  <https://docs.zammad.org/en/latest/admin/console.html>`_.
```

Comment:
After I read this note, I didn't understand

* which commands might differ from system to system,
* what systems these commands will work on, and
* how to modify these commands for other systems.

Without this information, this note raises more questions than it answers. I know the answers might be in the link, but I can't say for sure how that information relates to the code sample below.

I took a guess at what you meant and rewrote the note as follows:

```suggestion
.. note:: If you installed Zammad through a package manager (rather than from source),
   replace ``rails r`` with ``zammad run rails r`` below.
   To learn more, see `Administration via Console
   <https://docs.zammad.org/en/latest/admin/console.html>`_.
```

Did I get it right?

(Also, remember to use three-space indentation in rst directives like this. That way, the block matches up with the first line better.)